autoplex/data/rss/jobs.py

```diff
@@ -171,23 +255,41 @@ def _update_buildcell_options(self, updates, origin):
 
     def _cell_seed(
         self,
-        buildcell_options,
-        tag,
+        buildcell_parameters: list,
+        tag: str,
     ):
         """
-        Prepare random cells in self.directory.
+        Prepare the seed file for buildcell.
 
-        Arguments:
-        buildcell options :: (list of str) e.g. ['VARVOL=20']
+        Parameters
+        ----------
+        buildcell_parameters: (list of str) e.g. ['VARVOL=20']
+            List of parameters for creating the seed file for buildcell.
+        tag: str
+            Tag of systems.
         """
         bc_file = f"{tag}.cell"
         contents = []
-        contents.extend(["#" + i + "\n" for i in buildcell_options])
+        flag = False  # for printing blocks correctly with '#'
+        for i in buildcell_parameters:
+            if i.startswith("%") or flag:
+                flag = not (flag and i.startswith("%"))
+                contents.append(i + "\n")
+            else:
+                contents.append("#" + i + "\n")
 
         with open(bc_file, "w") as f:
             f.writelines(contents)
 
-    def _is_metal(self, element_symbol):
+    def _is_metal(self, element_symbol: str) -> bool:
```

Comment:
Wasn't it agreed to use pymatgen species for this part in your previous PR to reduce code duplicate codes? Did something change in from #114 PR and here that we explicitly need this ?

Reply:
Thank you for the reminder. I have applied the pymatgen function in the new code.